honeybee_3dm/helper.py

```diff
@@ -1,4 +1,41 @@
 """A collection of helper functions."""
+import json
+from enum import Enum, unique
+
+@unique
+class HB_layers(Enum):
+    """Default Honeybee layers.
+    
+    This class hosts all the default Honeybee layers in
+    layer key : layer name structure
+
+    Args:
+        Enum: An Enum object.
```

Comment:
I'm not sure if this is really helpful. Also Enum is the baseclass and not an argument.